mysqld_exporter.go

```diff
@@ -80,6 +92,21 @@ var (
 		"The total number of table I/O wait events for each table and operation.",
 		[]string{"schema", "name", "operation"}, nil,
 	)
+	performanceSchemaTableWaitsTimeDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, performanceSchema, "table_io_waits_picoseconds_total"),
+		"The total time of table I/O wait events for each table and operation.",
+		[]string{"schema", "name", "operation"}, nil,
+	)
+	performanceSchemaIndexWaitsDesc = prometheus.NewDesc(
```

Comment:
This and below form a summary. Are you sure you want to add a 3rd time unit to this exporter?